### PR TITLE
🚀 Add random method to Character

### LINKED ISSUE #448: Add extension to generate random character
Add an extension to `Character` to generate a random character

### PR DESCRIPTION
**Pull request by oliviabrown9**
Resolve #448

## Checklist

- [x] I checked the [**Contributing Guidelines**](https://github.com/SwifterSwift/SwifterSwift/blob/master/CONTRIBUTING.md) before creating this request.
- [x] New extensions are written in Swift 4.
- [x] New extensions support iOS 8.0+ / tvOS 9.0+ / macOS 10.10+ / watchOS 2.0+.
- [x] I have added tests for new extensions, and they passed.
- [x] All extensions have a **clear** comments explaining their functionality, all parameters and return type in English.
- [x] All extensions are declared as **public**.
- [x] I have added a [changelog](https://github.com/SwifterSwift/SwifterSwift/blob/master/CHANGELOG_GUIDELINES.md) entry describing my changes.

### CODE REVIEW

File: Tests/SwiftStdlibTests/CharacterExtensionsTests.swift [hunk 1]
```diff
@@ -68,6 +68,13 @@ final class CharacterExtensionsTests: XCTestCase {
     func testLowercased() {
         XCTAssertEqual(Character("S").lowercased, Character("s"))
     }
+    
+    func testRandom() {
+        let char1 = Character.random()
+        let char2 = Character.random()
+        
+        XCTAssertNotEqual(char1, char2)
```
Comment (calebkleveter) [inline Tests/SwiftStdlibTests/CharacterExtensionsTests.swift]: I understand the point of the test working this way, but it also causes to to randomly fail 🙂.

Comment (SD10) [reply Tests/SwiftStdlibTests/CharacterExtensionsTests.swift]: A better way to do this is to accumulate 2 separate strings using `Character.random()` and then compare the two. The longer the string the less likely a collision.

**Comment by oliviabrown9** [comment]
Updated to compare strings constructed of 10 Character.random() characters. Chose 10 as this is the length used for the random string tests, but it is otherwise arbitrary.

**Comment by oliviabrown9** [comment]
Updated name and removed the blank line SwiftLint warnings. Tangentially, removing some of the blank lines breaks the style of the file (consistent spacing between functions), so it might be beneficial to come up with a standard for whether blank lines are appropriate and enforce even retroactively.

**Comment by SD10** [comment]
Ah, @oliviabrown9 the warning was actually regarding an annoying Xcode default. It’s not warning against new lines but rather new lines that contain other forms of white space such as spaces or tabs. In your Xcode preferences you can actually set a flag to automatically trim the trailing whitespace. I wish this was the default ☹️

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by SD10** [comment]
Thank you for contributing to SwifterSwift! I've invited you to join the SwifterSwift GitHub organization - no pressure to accept! If you'd like more information on what that means, check out our [contributing guidelines](https://github.com/SwifterSwift/SwifterSwift/blob/master/CONTRIBUTING.md). Feel free to reach out if you have any questions! 😃

**Pull request merged**